ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java

```diff
@@ -57,7 +58,7 @@ public void addPatientAppointmentTest() throws Exception {
         findPatientPage = manageAppointmentsPage.saveAppointment();
         findPatientPage.enterPatient(patient.getName());
         manageAppointmentsPage = findPatientPage.clickOnFirstPatientAppointment();
```

Comment:
Thanks @insookwa , Seems @kdaud  finished this same work and it was merged here https://github.com/openmrs/openmrs-distro-referenceapplication/commit/336813b1c8a9bd44239a0ba4f37a2702834f7aa1  . Probably this is a duplicate. Is there any difference.

Reply:
AddPatientAppointment shares some logical steps with BookRequestAppointment test  though they both achive a different goal